recipes/newrelic/infrastructure/super-agent/debian.yml

```diff
@@ -390,6 +391,14 @@ install:
             sed -i '/^\s*#\s*agent_type: "newrelic\/io\.opentelemetry\.collector/s/#//' /etc/newrelic-super-agent/config.yaml
           fi
 
+    config_fleet_id:
+      cmds:
+        - |
+          if [ ! -z "{{.NR_CLI_FLEET_ID}}" ] ; then
+            sed -i 's/s*#\s*fleet_id:/fleet_id:/g' /etc/newrelic-super-agent/config.yaml
```

Comment:
Is this regexp ☝️  correct? I would say that the first `s` lacks a `\`

Reply:
(Same for the rest below)

Reply:
good catch, yes that was totally not ineffective